modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/Apply.kt

```diff
@@ -0,0 +1,264 @@
+@file:Suppress("UNUSED_PARAMETER")
+
+package arrow.typeclasses
+
+import arrow.Kind
+import arrow.core.*
+import java.math.BigDecimal
```

Comment:
This import seems weird, is it used anywhere?

Reply:
You are right, I have pending the imports optimization and the code formatting for this PR.